### PR TITLE
FCVendor

### PR DESCRIPTION
**Pull request by Sibg**
## About The Pull Request
The FC gets a mappable vendor with goodies for them!

## Why It's Good For The Game

Because FC's should get vendor? ;-; 

## Changelog
:cl:
add: The FC vendor
add: The FC now has its own access
/:cl:

### CODE REVIEW

**Comment by Sibg** [comment]
Diffs bot wont fix its shit, just ignore him i guess

**Comment by psykzz** [comment]
There are no icon changes anyway, so it wont show anything.

**Comment by psykzz** [comment]
Would you mind instead making a joint command staff vendor?

If you want to get smart you vend "cloaks" that auto dispense the correct cloak for the user, (we could add a boring cloak for staff or something).

Additionally this needs to be mapped in before we merge.

File: code/__DEFINES/access.dm [hunk 1]
```diff
@@ -19,6 +19,7 @@ GLOBAL_LIST_EMPTY(all_req_one_access)
 #define ACCESS_MARINE_SPECPREP		13
 #define ACCESS_MARINE_RESEARCH		14
 #define ACCESS_MARINE_SMARTPREP		15
+#define ACCESS_MARINE_COMMANDER		15
```
Comment (psykzz) [inline code/__DEFINES/access.dm:22]:
```suggestion
#define ACCESS_MARINE_COMMANDER		16
```

**Pull request merged**